sos/plugins/rhv_analyzer.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2018 Red Hat, Inc.
+#
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin
+
+
+class RhvLogCollectorAnalyzer(Plugin, RedHatPlugin):
+    """RHV Log Collector Analyzer"""
+
+    packages = ('rhv-log-collector-analyzer',)
+
+    plugin_name = 'rhv_analyzer'
```

Comment:
Your plugin name needs to match the lower-case class name: either "RhvLogCollectorAnalyzer" and "rhvlogcollectoranalyzer", or "Rhv_Analyzer" and "rhv_analyzer".